public/locales/zh.json

```diff
@@ -0,0 +1,1003 @@
+{
+    "navigation": {
+        "home": "navigation.home = 首页",
```

Comment:
```suggestion
        "home": "首页",
```

Reply:
Hi, I've committed the suggested changes. Please let me know if there's anything else that needs to be updated. Thank you for your guidance!